contracts/MarginBase.sol

```diff
@@ -342,11 +349,7 @@ contract MarginBase is MinimalProxyable, IMarginBase, OpsReady {
     ) internal returns (uint256 sizeDeltaInUSD) {
         /// @notice _sizeDelta is measured in the underlying base asset of the market
         /// @dev fee will be measured in sUSD, thus exchange rate is needed
-        sizeDeltaInUSD = exchangeRates().effectiveValue(
-            _market.baseAsset(),
-            _abs(_sizeDelta),
-            SUSD
-        );
+        sizeDeltaInUSD = (sUSDRate(_market) * _abs(_sizeDelta)) / 1 ether;
```

Comment:
np: I like dividing by `1e18` instead of `1 ether` that way it's more clear we're removing decimals.

Reply:
also if my order of operations in my head is correct i don't think you need parenthesis

Reply:
(1) Okay, will change!
(2) parenthesis are added when I format the document. You are right though, not needed.